gradle.properties

```diff
@@ -22,4 +22,7 @@ kotlin.code.style=official
 
 RELEASE_REPOSITORY_URL https://nexus.iroha.tech/repository/maven-soramitsu/
 SNAPSHOT_REPOSITORY_URL https://nexus.iroha.tech/repository/maven-soramitsu/
+android.defaults.buildfeatures.buildconfig=true
+android.nonTransitiveRClass=false
```

Comment:
I know that in new version of Gradle `nonTransitiveRClass=true` by default. But why do we need keep the old version of `nonTransitiveRClass=false`? `nonTransitiveRClass=true` can expedite app compilation